### ISSUE TITLE
DD4C Migration to Mauro

### ISSUE THREAD

**Issue by GSChandel** [NHSDigital/DataDictionaryPublication#242]
DD4C is currently hosted on infrastructure that is reaching end of Life.

The high-level need is to migrate the DD4C data to the Mauro supported platform. 

This need will involve investigation of the current DD4C database, creation of a new Mauro model and a migration process that takes copy of data from DD4C, transforms, and loads into Mauro.  

Who can access and any needs for future DD4C data amendments will need to be determined. 

Key assumptions are that data will be accessed via Mauro. (i.e. No publishing to Web involved) and that base Mauro functionality will provide any change functionality. (i.e. not intending to use the new Change functionality proposed for the Data Dictionary)

**Comment by oughnic** [comment]
I don't agree with the key assumption "No publishing to Web involved" - I assume that DD4C will be an integral part of the DD and published through the DD channel(s).
I also assume we will use Mauro change management to understand how the content of the refsets / valuesets have changed and the impact of those changes on other models in Mauro.

**Comment by cach1** [comment]
I agree with @oughnic there needs to be visibility on a web page for people wanting to view the DD4C metadata - via the DD website seems sensible and I assume it will be glueable to the DD content via DITA.  The dictionary needs to be able to link to the appropriate refset for certain data items as well, so people can see the members of a refset (the 'viewable at' section of current DD4C - it would be elegant to be able to pull this direct from ontoserver but a link into SNOMED browser would do for a start

**Comment by oughnic** [comment]
Thanks @cach1 - For now I think Mauro can be the source for code lists being surfaced.  The challenge is large refsets (all findings) so an arbitrary cut off for publication where big lists are not fully shared would be helpful.

**Comment by rajabr57** [comment]
6/5 - can be closed

**Comment by rajabr57** [comment]
23/6 - update from BC:  This is raised in on DP JIRA board and is in current sprint. Need to discuss with Ali.
